site/content/config.js

```diff
@@ -9,7 +9,10 @@ const userConfig = {
     text: "Life Itself",
   },
   navLinks: [
-    { href: "/about", name: "About" },
+    { name: "About", subItems: [
```

Comment:
I really don't want dropdowns atm - it's bad UX i think (plus the UI functionality here is a bit broken - which i will flag)

Reply:
Fixed 👍